src/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.html

```diff
@@ -111,62 +111,46 @@ <h2>NOTES</h2>
 
 <p>
 Sentinel products can be also filtered by <b>producttype</b> or, in case of
-Sentinel-2 data, and some of Sentinel-3, maximum <b>clouds</b> cover percentage.
-<!-- Add list of products here-->
+S2MSI1C, S2MSI2A, S3SY2SYN, S3SY2VGP, S3SY2VG1, S3SY2VG1 and S3SY2AOD, maximum <b>clouds</b> cover percentage.
 
 <p>
 Extra search keywords can be specified with <b>query</b>. Multiple keywords
-can be listed separated with comma (e.g. 'polarisationmode=VV,orbitdirection=Ascending').
-Note that ESA maintains a
-<a href="https://scihub.copernicus.eu/twiki/do/view/SciHubUserGuide/FullTextSearch">list of valid search keywords</a>
-that can be used with <b>query</b>. Usage of the <b>query</b> option for searching
-and downloading from the <b>USGS Earth Explorer</b> allows the keywords
-<i>identifier</i> and <i>filename</i>, both in ESA format. Additionally,
-the keyword <i>usgs_identifier</i> can be used to search for a specific scene
-in USGS name format (see <a href="https://lta.cr.usgs.gov/DD/Sentinel2.html#entity_id">USGS Entity ID</a>)
+can be listed separated with comma (e.g. 'orbitdirection=ASCENDING,instrument=SYNERGY').
+
+<b>List of Queryables:</b>
+<ul>
+  <li>orbitNumber</li>
+  <li>timeliness (e.g. NT)</li>
+  <li>processingBaseline</li>
+  <li>storageStatus</li>
+</ul>
+
+Note that text based queryables have to be upper-case.
+Scenes with unavailbe queryable information are filtered out
+(e.g. if a product have the timeliness queryable set as <b>Null</b>,
+then it will be filtered out if you use query="timeliness=NT").
+
 
 <p>
-<!-- Leave a note about uuid deprecation -->
 <em>i.sentinel.download</em> also allows downloading of Sentinel products
-by specifying a (list of) ID. This operation is performed by the <b>id</b>
+by specifying a (list of) ID, where ID refers to the scene name on
+Copernicus Data Space Ecosystem. This operation is performed by the <b>id</b>
 option. Note that this option is mutually exclusive with all other
 filtering options.
 
-<!-- Remove -->
 <p>
 In case a Sentinel data download was interrupted, <em>i.sentinel.download</em>
-will continue to download once the command is restarted and the partially
-downloaded file (with <tt>.incomplete</tt> file extention) still present.
+will restart the download for the paritally downloaded data, from scratch,
+once the command is re-excuted.
 
 <p>
 The <b>output</b> directory is created if not yet available. Default is current working directory.
 
-<p>
-The query string sent to ESA can be shown with <b>--verbose</b>.
-
 <!-- Not sure if Data Space Ecosystem doesn't have an archive, but apparently not -->
 <h3>Long Term Archive (DEPRECATED)</h3>
-Access to the product URL for data that are no longer available online in the Copernicus Data Space Ecosystem will automatically trigger
```

Comment:
Data on CSDE is "Immediately available data (IAD)". Ao no archive. Not sure about other providers. You can remove the entire section I guess, but I would mention the providers used somewhere...

Reply:
> but I would mention the providers used somewhere...

CDSE is the only provider used in the script currently, that is what you mean right?

Reply:
> > but I would mention the providers used somewhere...
> 
> CDSE is the only provider used in the script currently, this is what you mean right?